README.md

```diff
@@ -0,0 +1,31 @@
+Kuali Rice
+==========
+Kuali Rice is an application development framework and middle ware suite developed for and by the higher education community.  Kuali Rice is being used by several Kuali universiy enterprise applications, for its build in eDocLite document routing, and as basis for custom applications.
+
+More info about Kuali Rice at http://www.kuali.org/rice.
+
+Where to find the official version?
+-----------------------------------
+Endusers can download Kuali Rice at http://www.kuali.org/download/rice
+
+Developers can contribute to Kuali Rice at https://github.com/kuali/rice
+
+Additional Resources
+--------------------
+* Wiki:https://wiki.kuali.org/display/KULRICE/Home
+* Issue tracking: https://github.com/kuali/rice/issues
+* Technical help and discussions: rice.collab@kuali.org email list 
+  * Archive: https://groups.google.com/a/kuali.org/forum/#!forum/rice.collab
+
+Contributions
+-------------
+Contributions are welcome. The Rice Project strives for a simple and quick contribution process.  However, there are some specific requirements we need to enforce:
+* Please report all security issues to security@kuali.org. 
+* It is recommended to contact the Kuali Rice development team before starting work on a pull request.  Send the contribution proposal to the collaboration mailing list (rice.collab@kuali.org). The Kuali Rice team will evaluate the proposal, assign a Jira issue, and respond via email.
+* Every contributor must have a Contributor License Agreement (CLA on File with Kuali).  To submit the CLA:
+  * Create an account at http://kis.kuali.org/ (details: [Create a KIS account] (https://wiki.kuali.org/display/KULFOUND/Create+a+KIS+account)).
+  * Sign the [Contributor License Agreement] (https://docs.google.com/a/kuali.org/file/d/0B98dhdqKa_2hNzk5MzhhODgtYzM2Ni00MmUyLWFlYzYtNmJhMDdkZGM2MTI4/edit), scan and send it to help@kuali.org (details: [License Agreement Process (CLA / CCLA)] (https://wiki.kuali.org/pages/viewpage.action?pageId=313879878)).
+* Create a separate pull request for each feature.
+* Squash pull request into one commit.
```

Comment:
Can we provide a link or something with information on how to "squash" a pull request into one commit?